apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryInsertResults.cs

```diff
@@ -135,14 +136,17 @@ public BigQueryInsertResults ThrowOnAnyError()
                 return this;
             }
 
-            var exception = new GoogleApiException(_client.Service.Name, $"Error inserting data. Status: {Status}")
+            var flattenedErrors = Errors.SelectMany(rowErrors => rowErrors).ToList();
```

Comment:
At some point we should change Errors to an automatically-implemented property and remove _errors, so that we don't access them inconsistently. Probably best to do that in a separate PR though. (And I can do it.)

Reply:
Acked, happy to do that when I'm back